spec/requests/capabilities/README.md

```diff
@@ -0,0 +1,36 @@
+# Capabilities spec
+
+Goals of capabilities specs is to test capabilities only.
+
+- should test capabilities
+- should test capabilities for list and details endpoints
+  - lists: index, filter
+  - details: show, new, edit, update
```

Comment:
I assume this means it will return the capabilities with these routes? Otherwise if it only tests these routes, `delete` is missing

Reply:
doesn't make sense for `delete` since as far as I recall delete returns an empty response. What capabilities do you have with an empty response? Which actions do we need to expose there for UIs that aren't surfaced on the `list` page that users will be invariable redirected to next